doc/content/gateways/models/dragino-lps8/udp.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Connect Dragino LPS8N with UDP Packet Forwarder"
+description: ""
+---
+
+This section contains instructions for connecting the Dragino LPS8N LoRaWAN® gateway to {{% tts %}} using the [{{% udp-pf %}}]({{< ref "/gateways/concepts/udp" >}}).
+
+<!--more-->
+
+Before proceeding with the next step, we assume you have set up internet access for the LPS8N using one of the following methods:
+
+- Using the WAN port of the LPS8N
+- As a WiFi client
+- Using the built-in 4G modem
```

Comment:
Since this seems to be a prerequisite for both LBS and UDP modes, put this in the main file as part of prerequisites